<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationDialog</class>
 <widget class="QDialog" name="ApplicationDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление нового приложения</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Сектор:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="MyComboBox" name="secCombo" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Договор:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="6">
    <widget class="MyComboBox" name="dealCombo" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Тип приложения:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="6">
    <widget class="MyComboBox" name="typeCombo" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Наименование:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="QLineEdit" name="applicationEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Ссылка:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QLineEdit" name="linkEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="selectFileButton">
     <property name="toolTip">
      <string>Выбрать файл</string>
     </property>
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="winScanButton">
     <property name="toolTip">
      <string>Открыть WinScan2PDF для сканирования файла</string>
     </property>
     <property name="text">
      <string>WinScan2PDF</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QPushButton" name="saveButton">
     <property name="toolTip">
      <string>Сохранить приложение</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="3">
    <widget class="QPushButton" name="rejectButton">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QWidget</extends>
   <header location="global">mycombobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
